node/overseer/orchestra/src/lib.rs

```diff
@@ -14,29 +14,31 @@
 // You should have received a copy of the GNU General Public License
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
 
-//! # Overseer
+//! # Orchestra
 //!
-//! `overseer` implements the Overseer architecture described in the
-//! [implementers-guide](https://w3f.github.io/parachain-implementers-guide/node/index.html).
-//! For the motivations behind implementing the overseer itself you should
-//! check out that guide, documentation in this crate will be mostly discussing
-//! technical stuff.
+//! `orchestra` provides a global information flow of what a token of information.
```

Comment:
nit: maybe it is a good idea to mention that it was previously called `overseer` to help transition?

Reply:
Technically the `overseer` still exists as `polkadot-overseer` - only the _inner_ naming changed and I wanted to remove the polkadot references from the `orchestra/*` folders.